account.go

```diff
@@ -95,15 +95,32 @@ func (a AccountKey) Encode() []byte {
 	return mustRLPEncode(&temp)
 }
 
+// accountCompatibleAlgorithms returns true if the signature and hash algorithms are a valid pair
+// for a key on a Flow account.
+func accountCompatibleAlgorithms(sigAlgo crypto.SignatureAlgorithm, hashAlgo crypto.HashAlgorithm) bool {
+	switch sigAlgo {
+	case crypto.ECDSA_P256:
+		fallthrough
+	case crypto.ECDSA_secp256k1:
+		switch hashAlgo {
+		case crypto.SHA2_256:
+			fallthrough
```

Comment:
nit: isn't this a bit confusing? although it's technically correct in this case it just replaces the `return true` in the next case and I wonder if that makes it better? I feel like it would be more clear if it would just return true.

Reply:
I totally agree 😄 the `fallthrough` was in the original code and I kept it. I'll update it.